sematic/ui/src/components/RunPanel.tsx

```diff
@@ -106,6 +106,15 @@ export default function RunPanel(props: {
                 <Typography variant="h6">{selectedRun.name}</Typography>
```

Comment:
Can we also add something to the logs view that says "No logs because this was cloned from X"? Or something along those lines?

Reply:
Done.